From ae0a2b93af52b98eeabf6b37cbbcb89dba1ae1ce Mon Sep 17 00:00:00 2001
From: Hains van den Bosch <hainsvdbosch@ziggo.nl>
Date: Thu, 30 May 2019 17:08:48 +0200
Subject: [PATCH] Revert "startwizard.xml: Add PESpeedUp"

As we're not allowed to use PESpeedUP with unofficial images.

This reverts commit e47ae6e6094cc3f096fb05c37ceea8a37e4814ea.
---
 data/startwizard.xml | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/data/startwizard.xml b/data/startwizard.xml
index 628f5923f..943c972af 100644
--- a/data/startwizard.xml
+++ b/data/startwizard.xml
@@ -11,7 +11,6 @@ self.selectKey("OK")
 			<list>
 				<listentry caption="Use the wizard to set up basic features" step="network"/>
 				<listentry caption="Exit the wizard" step="end"/>
-				<listentry caption="Speed up the performance" step="pespeedup" />
 			</list>
 			<code>
 self.clearSelectedKeys()
@@ -19,12 +18,6 @@ self.selectKey("UP")
 self.selectKey("DOWN")
 			</code>
 		</step>
-		<step id="pespeedup">
-			<code>
-from Plugins.Extensions.PESpeedUp.plugin import *
-self.session.open(PESpeedUp)
-			</code>
-		</step>
 		<step id="network" >
 			<text value="Please connect your receiver to the internet"/>
 			<list>
-- 
2.20.1

